### DISCUSSION TITLE
Thorium Legacy 124?

### DISCUSSION THREAD

**Discussion by DaveH1** [Alex313031/thorium-legacy#102] [Ideas]
Is it in the pipeline?

**Comment by gz83** [comment]
Please wait patiently until Alex has time to develop

**Comment by DaveH1** [comment]
Don't worry, I will be patient!
I just wanted to ask if it was going to be done, as Supermium has had a 124 version for Windows XP out for a while now.
Thorium Legacy 122 is still working fine, so there's no rush!
Cheers, Dave.

**Reply by leao-nardo** [reply]
i would ask , what is this supermium ?
seems like this https://github.com/win32ss/supermium
and theres v13 already ,great!
